geomesa-bigtable/pom.xml

```diff
@@ -49,11 +36,11 @@
             <id>bigtable</id>
             <modules>
                 <module>geomesa-bigtable-datastore</module>
-                <module>geomesa-bigtable-dist</module>
+                <!--<module>geomesa-bigtable-dist</module>-->
                 <module>geomesa-bigtable-gs-plugin</module>
                 <module>geomesa-bigtable-tools</module>
-                <module>geomesa-bigtable-spark</module>
-                <module>geomesa-bigtable-spark-runtime</module>
+                <!--<module>geomesa-bigtable-spark</module>-->
+                <!--<module>geomesa-bigtable-spark-runtime</module>-->
```

Comment:
Thanks for the submission! Do these modules no longer work, or were they commented out accidentally?

Reply:
Oh, I was just having trouble rebuilding this locally so I commented them out. Will fix now.